### PR TITLE
DROOLS-2393: [DMN Designer] Not expected results of commands when multiple cells selected

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-2393

### CODE REVIEW

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/types/context/ContextGridTest.java [hunk 1]
```diff
@@ -401,33 +401,74 @@ public void testOnItemSelectedExpressionColumnDefinedExpressionType() {
...
+    @Test
+    public void testGetItemsWithCellSelectionsCoveringMultipleRows() {
+        setupGrid(0);
+
+        addContextEntry(0);
+        grid.getModel().selectCell(0, 0);
+        grid.getModel().selectCell(1, 0);
+
+        assertDefaultListItems(grid.getItems(0, 0), false);
+    }
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/types/context/ContextGridTest.java]: I think I miss one more basic scenario:
```java
@Test
public void testGetItemsWithCellSelectionsCoveringMultipleColumns() {
    setupGrid(0);

    grid.getModel().selectCell(0, 0);
    grid.getModel().selectCell(0, 1);

    assertDefaultListItems(grid.getItems(0, 0), true);
}
```

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/widgets/panel/DMNGridPanelCellSelectionHandlerImpl.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            final LiteralExpressionGrid grid = (LiteralExpressionGrid) _gridWidget;
+            _gridWidget = grid.getParentInformation().getGridWidget();
+            _uiRowIndex = grid.getParentInformation().getRowIndex();
+            _uiColumnIndex = grid.getParentInformation().getColumnIndex();
+        } else if (_gridWidget instanceof UndefinedExpressionGrid) {
+            final UndefinedExpressionGrid grid = (UndefinedExpressionGrid) _gridWidget;
+            _gridWidget = grid.getParentInformation().getGridWidget();
+            _uiRowIndex = grid.getParentInformation().getRowIndex();
+            _uiColumnIndex = grid.getParentInformation().getColumnIndex();
+        }
```
[the remark this answers is not in the document]
Comment (manstis) [reply kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/widgets/panel/DMNGridPanelCellSelectionHandlerImpl.java]: Changed in new commit.

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/types/context/ContextGridTest.java (hunk 1 above)
Comment (manstis) [reply kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/types/context/ContextGridTest.java]: Added in new commit.

**Comment by manstis** [comment]
@jomarko Changes made in new commit.

**Pull request merged**